sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -1143,6 +1134,11 @@ public abstract partial class ObjectSerializer
         public abstract System.Threading.Tasks.ValueTask SerializeAsync(System.IO.Stream stream, object? value, System.Type inputType, System.Threading.CancellationToken cancellationToken);
         public virtual System.Threading.Tasks.ValueTask<System.BinaryData> SerializeAsync(object? value, System.Type? inputType = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
+    public enum PropertyNameHandling
+    {
+        Strict = 0,
+        ToCamelCase = 1,
```

Comment:
For context, the `System.Text.Json.Serialization` namespace has a [JsonKnownNamingPolicy](https://learn.microsoft.com/en-us/dotnet/api/system.text.json.serialization.jsonknownnamingpolicy?view=net-8.0) enum that corresponds to this.  I think this name is too long for our purposes, but it is interesting that the functional equivalent exists in STJ.